test/long-stack-traces.js

```diff
@@ -8,13 +8,13 @@ import {join, fixtureDir} from './_utils';
 
 function internals() {
 	return StackUtils.nodeInternals().concat([
-		/[\\\/]long-stack-traces\.js:[0-9]+:[0-9]+\)?$/,
-		/[\\\/]internal-error\.js:[0-9]+:[0-9]+\)?$/,
-		/[\\\/]internal-then\.js:[0-9]+:[0-9]+\)?$/,
-		/[\\\/]node_modules[\\\/]/,
+		/\/long-stack-traces\.js:[0-9]+:[0-9]+\)?$/,
```

Comment:
I'm aware. `[0-9]` is the way they were expressed in `node-tap` originally. I kinda like it better than the shortcut anyways. I think it's easier to read, especially with the amount of escaping going on around it.